Projects/SvitlanaLN/STAR_3_Figure/Figure.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System;
+using System.Globalization;
+
+namespace Star_3_Figure
+{
+    [Serializable]
+    public class Figure
+    {
+        //перевіряю, чи дата коректна 
+        public static void Correct_Parse_DateTime(string inputString)
+        {
+            if (!DateTime.TryParse(inputString, out DateTime _1))
+                throw new ArgumentException();
+            else if ((DateTime.TryParse(inputString, out DateTime _2))&(_2 >DateTime.Today.Date))
+                throw new ArgumentOutOfRangeException();
+        }
+
+        //зчитуємо дані, поки дата не буде коректною
```

Comment:
Please rename methods. Each method name should contain a verb.

Reply:
Done.